hackathon_site/dashboard/frontend/src/pages/Teams/Teams.tsx

```diff
@@ -2,13 +2,43 @@ import React from "react";
 // import styles from "./Teams.module.scss";
 import Header from "components/general/Header/Header";
 import Typography from "@material-ui/core/Typography";
+import Grid from "@material-ui/core/Grid";
+import styles from "pages/Teams/Teams.module.scss";
+
+import TeamCardAdmin from "components/dashboard/TeamCardAdmin/TeamCardAdmin";
+import InventorySearch from "components/inventory/InventorySearch/InventorySearch";
+import { teamsList } from "testing/mockData";
+
+// import TeamCardAdmin from "../../components/dashboard/TeamCardAdmin/TeamCardAdmin";
 
 const Teams = () => {
+    const CardComponents = teamsList.map((team) => (
+        <Grid
+            item
+            md={3}
+            sm={4}
+            xs={6}
```

Comment:
change to `xs={12}`, it gets very squished on mobile

Reply:
changed to 12